packages/mobile/public/index.html

```diff
@@ -5,7 +5,7 @@
             http-equiv="Content-Security-Policy"
             content="
                 default-src 'self';
-                connect-src 'self' https://*;
+                connect-src 'self' https://* ws://*;
```

Comment:
This change is needed to test iOS

Reply:
is the code needed for testing purposes only, or does it need to be added on the production code? If its only for testing, I would add this as an intruction in the PR and remove this line, otherwise we can resolve this comment

Reply:
I've just removed it and added an issue here: https://github.com/iotaledger/firefly/issues/6346